cirq/circuits/text_diagram_drawer.py

```diff
@@ -83,14 +83,15 @@ def __init__(self,
                 else dict(vertical_padding)
                 ) # type: Dict[int, Union[int, float]]
 
+    def _value_equality_values_(self):
+        attrs = ('entries', 'horizontal_lines', 'vertical_lines',
+                 'horizontal_padding', 'vertical_padding')
+        return tuple(getattr(self, attr) for attr in attrs)
+
     def __eq__(self, other):
```

Comment:
Delete the `__eq__` method when using value equality.

Reply:
Yea, that's just temporary because the class isn't actually dressed with `value_equality` yet.

Reply:
We need to resolve the dependency cycle so it can be annotated with it.

Reply:
Done